WebARonARKit/WebARonARKit.js

```diff
@@ -874,17 +874,60 @@
     return this;
   };
 
+  /**
+   * The structure that represents a plane detected by the underlying AR
+   * system.
+   * This structure has 4 properties: 
+   * - identifier: The unique id of the plane.
+   * - modelMatrix: a matrix that represents the position and orientation
+   * of the plane in world space.
+   * - extent: The 2 dimensions extents of the plane.
+   * - vertices: The list of 3D points that conform the plane.
+   * @constructor
+   */
+  VRPlane = function(plane) {
+    this.set_(plane);
+    return this;
+  };
+
+  VRPlane.prototype.set_ = function(plane) {
+    this.identifier = plane.identifier;
+    this.modelMatrix = new Float32Array(plane.modelMatrix.length);
+    for (var i = 0; i < plane.modelMatrix.length; i++) {
+      this.modelMatrix[i] = plane.modelMatrix[i];
+    }
+    this.extent = new Float32Array(plane.extent.length);
+    for (var i = 0; i < plane.extent.length; i++) {
+      this.extent[i] = plane.extent[i];
+    }
+    this.vertices = new Float32Array(plane.vertices.length);
+    for (var i = 0; i < plane.vertices.length; i++) {
+      this.vertices[i] = plane.vertices[i];
+    }
+    return this;
+  };
+
+  /**
+   * The structure that represents an anchor created by the user and notified
+   * to the underlying AR platform so it keeps track of it. If the AR system
+   * changes its tracking estimation, the user anchors might need to be updated
+   * too.
+   * This structure has 2 properties: 
+   * - identifier: The unique id of the anchor.
+   * - modelMatrix: a matrix that represents the position and orientation
+   * of the anchor in world space.
+   * @constructor
+   */
   VRAnchor = function(modelMatrix) {
     this.identifier = WebARonARKitVRDisplay.anchorsIdCounter_++;
     this.modelMatrix = new Float32Array(16);
     if (modelMatrix && modelMatrix.length === 16) {
       for (var i = 0; i < 16; i++) {
         this.modelMatrix[i] = modelMatrix[i];
       }
-    }
-    else {
+    } else {
       throw "ERROR: The given parameter is not a 16 value matrix in the " +
-          "VRAnchor constructor.";
+      "VRAnchor constructor.";
```

Comment:
this indentation is wrong - it was correct before (4 space continuation indent)

Reply:
You are right. Fixed.